app/controllers/declaration/TransportDetailsController.scala

```diff
@@ -56,14 +60,20 @@ class TransportDetailsController @Inject()(
       .fold(
         (formWithErrors: Form[TransportDetails]) => Future.successful(BadRequest(transport_details(formWithErrors))),
         transportDetails =>
-          customsCacheService
-            .cache[TransportDetails](cacheId, TransportDetails.formId, transportDetails)
-            .map(_ => redirect(transportDetails))
+          for {
+            _ <- updateCache(journeySessionId, transportDetails)
+            _ <- customsCacheService.cache[TransportDetails](cacheId, TransportDetails.formId, transportDetails)
+          } yield redirect(transportDetails)
       )
   }
 
   private def redirect(transportDetails: TransportDetails)(implicit request: JourneyRequest[_]): Result =
     if (transportDetails.container) Redirect(TransportContainerController.displayPage())
     else if (request.choice.value == AllowedChoiceValues.StandardDec) Redirect(SealController.displayForm())
     else Redirect(SummaryPageController.displayPage())
+
+  private def updateCache(sessionId: String, formData: TransportDetails): Future[Either[String, ExportsCacheModel]] =
+    updateHeaderLevelCache(sessionId, model => {
```

Comment:
You don't need curly brackets here: `sessionId, model => {` it's comment for all similar methods.

Reply:
This has been addressed!

Reply:
This has been addressed